cupy/__init__.py

```diff
@@ -446,6 +446,7 @@ def base_repr(number, base=2, padding=0):  # NOQA (needed to avoid redefinition
 from cupy.linalg.einsum import einsum  # NOQA
 
 from cupy.linalg.product import dot  # NOQA
+from cupy.linalg.product import cross  # NOQA
```

Comment:
Oops, please order the imports alphabetically, as per the OpenStack style guideline.